### DISCUSSION TITLE
My Tailwind output file contains a lot of classes which I don't use

### DISCUSSION THREAD

**Discussion by Alice221299** [tailwindlabs/tailwindcss#12621] [Help]
The output file is 271351 lines long and mostly contains of classes like `.border-s-emerald-100\/20{border-inline-start-color: rgb(209 250 229 / 0.2);}` which I've never used in the code. It happened when I installed the latest 3.3.7 version and when I got back to 3.3.6 nothing happened. I don't know how to get rid of the unused classes. Please help

**Comment by wongjn** [comment]
Consider providing a remotely-hosted git repository that reproduces the unexpected behavior please.

**Reply by Alice221299** [reply]
Sadly, I can't because it belongs to a company

**Reply by wongjn** [reply]
It doesn't have to be the exact same one as where you are experiencing the problem. You could consider providing a different remotely-hosted git repository that reproduces the same unexpected behavior. This exercise may also help you to troubleshoot the problem.

**Comment by Alice221299** [comment]
It was the safelist part in config file which I didn't notice
